repos/spack_repo/builtin/packages/dbcsr/package.py

```diff
@@ -108,6 +108,12 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     dbcsr_amdgpu_targets = ("gfx906", "gfx910", "gfx90a", "gfx90a:xnack-", "gfx90a:xnack+")
     amd_msg = f"DBCSR supports these AMD gpu targets:  {', '.join(dbcsr_amdgpu_targets)}"
 
+    conflicts(
+        "+rocm",
+        when="amdgpu_target=gfx942",
+        msg="DBCSR does not support gfx942. Please use amdgpu_target=gfx90a",
+    )
+
```

Comment:
Actually, I did not realize this before from your PR, but I think the conflict below already takes care of this, since `gfx942` is not added to `dbcsr_amdgpu_targets`?